sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -91,32 +95,88 @@ credential.update("{new_api_key}");
 
 ## Key concepts
 ### Client
-The Form Recognizer client library provides a [FormRecognizerClient][form_recognizer_sync_client] and 
-[FormRecognizerAsyncClient][form_recognizer_async_client]. It provides both synchronous and
-asynchronous operations to access a specific use of Form Recognizer, such as training custom models,
-extracting layout and receipt information from documents.
+The Form Recognizer client library provides a [FormRecognizerClient][form_recognizer_sync_client], 
+[FormRecognizerAsyncClient][form_recognizer_async_client] and [FormTrainingClient][form_training_sync_client],
+[FormTrainingAsyncClient][form_training_async_client]. It also supports model management operations to
+get custom model info, delete model, list models and get account details.
+
+### FormRecognizerClient
+A `FormRecognizerClient` and `FormRecognizerAsynClient` provides both synchronous and asynchronous operations to access
+ a specific use of Form Recognizer, such as recognizing layout, receipt and custom form data from documents.
+<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L39-L42 -->
```

Comment:
Probably need to update this.
I saw in here you already have the readme sample.
https://github.com/Azure/azure-sdk-for-java/pull/10218/files#diff-49d7f9d8b6def8c99411771de4caf755L106